appinfo/info.xml

```diff
@@ -7,17 +7,14 @@
 
 		The application listens to the creation of new image files, and queues them for later analysis. A scheduled task (Or admin on demand) take this queue, and analyze the images for looking faces and if possible identify them by comparing them with those found in user-folder/.faces.]]>
 	</description>
-	<licence>AGPL</licence>
+	<version>0.5.7</version>
+	<licence>agpl</licence>
```

Comment:
for my own education, what was different between `AGPL` and `agpl`? It requires lower case?

Reply:
It's strange, but it was the report ... . :sweat_smile: 
```
Invalid appinfo.xml file found: Element 'licence': [facet 'enumeration'] The value 'AGPL' is not an element of the set {'agpl', 'mpl', 'apache'}.

Invalid appinfo.xml file found: Element 'licence': 'AGPL' is not a valid value of the atomic type 'licence'.

Invalid appinfo.xml file found: Element 'licence': Warning: No precomputed value available, the value was either invalid or something strange happend.
```